src/guide/reactivity-computed-watchers.md

```diff
@@ -234,6 +234,16 @@ setup() {
 }
 ```
 
+If you want watcher to be triggered after every change, make sure to add the [nextTick](/api/global-api.html#nexttick) call between changes:
+
+```js
+const changeValues = async () => {
+  firstName.value = 'John' // logs: ["John", ""] ["", ""]
+  await nextTick()
+  lastName.value = 'Smith' // logs: ["John", "Smith"] ["John", ""]
+}
```

Comment:
Now it's possible to execute `cb` twice without `await nextTick()`

Reply:
```ts
const firstName = ref('')
const lastName = ref('')

watch([firstName, lastName], (newValues, prevValues) => {
  console.log(newValues, prevValues)
})

firstName.value = 'John' // logs: ["John", ""] ["", ""]
lastName.value = 'Smith' // logs: ["John", "Smith"] ["John", ""]
```
I think not to add `await nextTick()` is in line with expectations, after all, the documentation is also written in this way at the beginning, there is no possibility that the problem is now inside the `vue code`?